.pylintrc

```diff
@@ -2,8 +2,7 @@
 ignore-patterns=__main__.py,.*_pb2.py,tac.sh,tac_local.sh
 
 [MESSAGES CONTROL]
-# Skips after C0209 are temporary
-disable=C0103,C0201,C0301,C0302,W0105,W0707,W1202,W1203,R0801,E1136,E0611,C0209,R1710,R1729,C0206,W0237,R1735,W1505,R1732,W1514,R0402,R1714,R1734,W0640,W0621,E0401,R1733
+disable=C0103,C0201,C0301,C0302,W0105,W0707,W1202,W1203,R0801,E1136,E0611,C0209
```

Comment:
Please remove C0209, it's the error code for not using the f-string formatting and since you've addressed this you can remove it

Reply:
resolved https://github.com/valory-xyz/open-aea/pull/365#discussion_r994409381

Reply:
No, I cannot remove it due to: https://github.com/valory-xyz/open-aea/pull/365#discussion_r994409381